recipe/meta.yaml

```
@@ -25,6 +25,7 @@ requirements:
     - cmake
     - ninja
     - pandoc
+    - pkgconfig
```

So this is a Python package called `pkgconfig`. This would be `import`ed and used that way

However it looks like `rdma-core` simply needs the CLI executable `pkg-config` (note `-`)

So would use that here instead

```suggestion
    - pkg-config
```

Side note:Guessing we only see this now as the previous Docker image had `pkg-config` on it already (likely as a transitive dependency of something else installed). Hence we now are needing to explicitly require it, which is better anyways for reproducibility